sdk/cosmosdb/cosmos/test/internal/unit/platform.spec.ts

```diff
@@ -1,15 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import assert from "assert";
-import { Constants } from "../../../dist-esm/index";
-import { getUserAgent } from "../../../dist-esm/common";
+import { Constants , getUserAgent} from "@azure/cosmos";
```

Comment:
```suggestion
import { Constants , getUserAgent} from "../../../../src";
```

Tests should import directly from source.

Reply:
Done